src/main/java/Edit_Group_Profile_Use_Case/EditGroupProfileInputBoundary.java

```diff
@@ -0,0 +1,14 @@
+package Edit_Group_Profile_Use_Case;
+import Entities.GroupProfile;
+
+
+public interface EditGroupProfileInputBoundary {
+    /**
+     * Triggers the EditGroupProfileUseCase.
+     * @param requestModel
+     * @return
+     */
+    EditGroupProfileResponseModel editGroup(EditGroupProfileRequestModel requestModel);
+
+}
```

Comment:
Good change in naming. It will make it a lot easier for the reader to know what is happenning. No more code smells!